### PR TITLE
Create early version of time management service

### PR DESCRIPTION
**Pull request by Taylorr82**
## Merge checklist
- [ ] Docker image or CCS Project builds with the changes
- [ ] Docker image or CCS Project runs with the changes
- [ ] Can send commands from a mocked ground station/relevant changes tested
- [ ] Changes have been reviewed

### CODE REVIEW

**Comment by Taylorr82** [comment]
Cannot merge until GPS software is added to main repo

File: Services/include/util/time_struct.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
```
Comment (arrooney) [inline Services/include/util/time_struct.h:1]: Put our licence

File: Services/source/time_management/time_management_service.c [hunk 2]
```diff
@@ -24,13 +24,57 @@
...
+
+#define GPS_TASK_SIZE 200
+#define NMEA_TASK_SIZE 100
+
+#define MIN_YEAR 1577836800  // 2020-01-01
+#define MAX_YEAR 1893456000  // 2030-01-01
 
 SAT_returnState time_management_app(csp_packet_t *packet);
 
+void RTC_discipline_service(void) {
```
Comment (arrooney) [inline Services/source/time_management/time_management_service.c:43]: comment this

File: Services/source/util/service_utilities.c [hunk 3]
```diff
@@ -141,6 +149,14 @@ void cnv8_D(uint8_t *from, double *to) {
   *to = cnv.cnvD;
 }
 
+uint16_t betole(uint16_t x) {
```
Comment (arrooney) [inline Services/source/util/service_utilities.c:152]: include "csp_endian.h" instead

**Pull request merged**